src/main/java/lotto/controller/Controller.java

```diff
@@ -0,0 +1,29 @@
+package lotto.controller;
+
+import lotto.domain.CheckPlace;
+import lotto.domain.CustomLotto;
+import lotto.domain.Reward;
+import lotto.domain.WinNums;
+import lotto.generator.NumberGenerator;
+import lotto.generator.RandomNumberGenerator;
+import lotto.domain.LottoGame;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+public class Controller {
+
+    private final NumberGenerator numberGenerator = new RandomNumberGenerator();
+
+    public void run() {
+        int inputMoney = InputView.getMoney();
+        int customCount = InputView.getCustomLottoCount();
+        CustomLotto customLotto = new CustomLotto(InputView.getCustomLotto(customCount));
+        LottoGame lottoGame = new LottoGame(inputMoney, numberGenerator, customLotto);
+        OutputView.printLottoes(customCount, lottoGame.getLottoList());
+        WinNums winNums = new WinNums(InputView.getWinLotto(), InputView.getBonusBall());
+        CheckPlace checkPlace = new CheckPlace(lottoGame, winNums);
+        OutputView.printStatistics(checkPlace.getResultMap());
+        Reward reward = new Reward(checkPlace.getResult(), inputMoney);
+        OutputView.printReward(reward.getRate());
+    }
```

Comment:
취향 차이라고 할 수 있는지 모르겠는데 저라면 중간 중간 가독성을 위해 개행을 넣었을 것 같아요~ 어떻게 생각하시나요?
```suggestion
    public void run() {
        int inputMoney = InputView.getMoney();
       
        int customCount = InputView.getCustomLottoCount();
        CustomLotto customLotto = new CustomLotto(InputView.getCustomLotto(customCount));
        
        LottoGame lottoGame = new LottoGame(inputMoney, numberGenerator, customLotto);
        OutputView.printLottoes(customCount, lottoGame.getLottoList());
        
        WinNums winNums = new WinNums(InputView.getWinLotto(), InputView.getBonusBall());
        CheckPlace checkPlace = new CheckPlace(lottoGame, winNums);
        OutputView.printStatistics(checkPlace.getResultMap());
        
        Reward reward = new Reward(checkPlace.getResult(), inputMoney);
        OutputView.printReward(reward.getRate());
    }
```

Reply:
개행하니 가독성이 더 좋아진것 같아요